sentry_sdk/integrations/django/signals_handlers.py

```diff
@@ -19,13 +19,17 @@ def _get_receiver_name(receiver):
     name = ""
 
     if hasattr(receiver, "__qualname__"):
-        name += receiver.__qualname__
+        name = receiver.__qualname__
     elif hasattr(receiver, "__name__"):  # Python 2.7 has no __qualname__
-        name += receiver.__name__
+        name = receiver.__name__
+    elif hasattr(
+        receiver, "func"
+    ):  # certain functions (like partials) dont have a name
+        name = "partial(<function " + receiver.func.__name__ + ">)"  # type: ignore
```

Comment:
can the func itself be a partial?

Reply:
F*ck... Probably... I will check on Monday

Reply:
I will check now.

Reply:
Ok, `func` appears to always be the function:

```
Python 3.10.8 (main, Oct 13 2022, 09:48:40) [Clang 14.0.0 (clang-1400.0.29.102)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> def f(a, b, c, x):
...     return 1000*a + 100*b + 10*c + x
... 
>>> f
<function f at 0x102752050>
>>> from functools import partial
>>> g = partial(f, 3)
>>> h = partial(g, 1)
>>> g
functools.partial(<function f at 0x102752050>, 3)
>>> h
functools.partial(<function f at 0x102752050>, 3, 1)
>>> h(3, 4)
3134
>>> h.func
<function f at 0x102752050>
>>> g.func
<function f at 0x102752050>
```

Reply:
cool, thx for checking